api/app/models/spree/webhooks/event.rb

```diff
@@ -0,0 +1,7 @@
+module Spree
+  module Webhooks
+    class Event < Spree::Base
+      self.table_name = 'spree_webhooks_events'
```

Comment:
@sebastian-palma I think we can skip table_name definitions - If I'm not mistaken it will be automatically resolved to proper table name based on the class name

Reply:
It won't work as is because it tries to look for a `spree_api_spree_test.spree_events` table. We could leave the tables like `spree_events` but I'm not sure if won't be a problem then.